sdk/cdsclient/client_user.go

```diff
@@ -72,6 +73,14 @@ func (c *client) UserGetSchema(ctx context.Context) (sdk.SchemaResponse, error)
 	return res, nil
 }
 
+func (c *client) UserGetSchemaV2(ctx context.Context, entityType string) (res sdk.Schema, err error) {
```

Comment:
why did you make a type sdk.Schema for just a []byte ?
IMO it's a bit misleading, at the first sight, i thought i was a structured type and this code doesn't have any sens  ><